demisto_sdk/commands/format/update_generic.py

```diff
@@ -255,7 +255,7 @@ def set_from_version_key_name(self) -> Union[str, None]:
     def is_old_file(path: str) -> dict:
         """Check whether the file is in git repo or new file.  """
         if path:
-            data = get_remote_file(path)
+            data = get_remote_file(path, suppress_print=True)
```

Comment:
why?

Reply:
We do not have a mechanism to identify pre-existing files from new ones in format - this prevents an unnecessary print in case this is a new file - I can keep it in verbose though - I'll fix it.